packages/khalti/test/src/khalti_http_client_test.dart

```diff
@@ -0,0 +1,259 @@
+import 'dart:convert';
+
+import 'package:http/http.dart' as http;
+import 'package:khalti/khalti.dart';
+import 'package:mocktail/mocktail.dart';
+import 'package:test/test.dart';
+
+typedef Caller = Future<HttpResponse> Function(String, Map<String, Object>);
+
+class _MockHttp extends Mock implements http.Client {}
+
+class _MockResponse extends Mock implements http.Response {}
+
+class _FakeUri extends Fake implements Uri {}
```

Comment:
Always add Mock, Fake at the end of sub-class name. There are two reasons for it. 
1. whenever you type the real class name, you'll see the mock together in the suggestion list
2. it's best practice to add base-class as a suffix for sub-class.